pandas/core/indexes/timedeltas.py

```diff
@@ -127,6 +141,15 @@ def _join_i8_wrapper(joinf, **kwargs):
 
     _freq = None
 
+    # TODO: Deduplicate with DatetimeIndex by doing these as props on base
+    _box_func = TimedeltaArray._box_func
+    _box_values = TimedeltaArray._box_values
+    _validate_frequency = TimedeltaArray._validate_frequency
```

Comment:
I'm pretty sure we dont need _validate_frequency on the Indexes anymore.  (might be future tense for DatetimeIndex)